### PR TITLE
Feat/svdb query

### PR DESCRIPTION
**Pull request by ramprasadn**
## PR checklist

This PR adds a subworkflow to annotate structural variants using svdb query

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
    - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/raredisease/tree/master/.github/CONTRIBUTING.md)
    - [ ] If necessary, also make a PR on the nf-core/raredisease _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker -stub`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: modules/nf-core/modules/svdb/query/main.nf [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    }
+    if (in_frqs) {
+        in_frq  = "--in_frq ${in_frqs.join(',')}"
+    }
+    if (out_occs) {
+        out_occ = "--out_occ ${out_occs.join(',')}"
+    }
+    if (out_frqs) {
+        out_frq = "--out_frq ${out_frqs.join(',')}"
+    }
```
Comment (jemten) [inline modules/nf-core/modules/svdb/query/main.nf:43]: This could be a nice case for the ternary operator

File: workflows/raredisease.nf [hunk 2]
```diff
@@ -72,6 +73,8 @@ include { ANNOTATE_VCFANNO            } from '../subworkflows/nf-core/annotate_v
 include { CALL_STRUCTURAL_VARIANTS    } from '../subworkflows/nf-core/call_structural_variants'
 
 
+include { ANNOTATE_STRUCTURAL_VARIANTS } from '../subworkflows/local/annotate_structural_variants'
```
Comment (jemten) [inline workflows/raredisease.nf:76]: local -> nf-core subworkflow

File: workflows/raredisease.nf [hunk 3]
```diff
@@ -155,6 +158,18 @@ workflow RAREDISEASE {
     )
     ch_versions = ch_versions.mix(CALL_STRUCTURAL_VARIANTS.out.versions)
 
+    ch_sv_annotate = Channel.empty()
+    if (params.svdb_query_dbs) {
```
Comment (jemten) [inline workflows/raredisease.nf:162]: Think about whether the switch should be on file or subworkflow/module level.

**Review by jemten — COMMENTED** [review]
The pipeline is getting closer to finished!

File: workflows/raredisease.nf (hunk 3 above)
Comment (ramprasadn) [reply workflows/raredisease.nf]: Good point! I have now moved the switch to the subworkflow level 😄

**Review by jemten — APPROVED** [review]
Looks good

**Pull request merged**